reference/commands/lock/add.rst

```diff
@@ -29,3 +29,89 @@ conan lock add
       --lockfile-out LOCKFILE_OUT
                             Filename of the created lockfile
       --lockfile LOCKFILE   Filename of the input lockfile
+
+The ``conan lock add`` command is able to add a package version to an existing or new lockfile ``requires``, ``build_requires`` or ``python_requires``.
+
+For example, the following is able to create the following lockfile (by default, named ``conan.lock``):
+
+.. code-block:: bash
+
+  $ conan lock add --requires=pkg/1.1 --build-requires=tool/2.2 --python-requires=mypytool/3.3 
+  Generated lockfile: ...conan.lock
+
+  $cat conan.lock
+  {
+      "version": "0.5",
+      "requires": [
+          "pkg/1.1"
+      ],
+      "build_requires": [
+          "tool/2.2"
+      ],
+      "python_requires": [
+          "mypytool/3.3"
+      ]
+  }
+
+
+If an input profile is provided, the command will add the arguments to the existing lockfile, and maintain the
```

Comment:
```suggestion
If an input lockfile is provided, the command will add the arguments to the existing lockfile, and maintain the
```

Reply:
But I think I would rephrase this to something like: "The conan lock add command also allows you to use an existing lockfile as an input...."